lib/internal/test_runner/tests_stream.js

```diff
@@ -57,6 +57,10 @@ class TestsStream extends Readable {
     this[kEmitMessage]('test:diagnostic', { __proto__: null, nesting, file, message });
   }
 
+  stdout(nesting, file, line) {
```

Comment:
Actually I just realized, it looks like `nesting` is always `0`.

Reply:
yes